get-modified-packages/README.md

```diff
@@ -4,13 +4,15 @@ This action get the list of ROS packages modified in the pull request.
 
 ## Usage
 
+### Basic
+
 ```yaml
 jobs:
   get-modified-packages:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
```

Comment:
```suggestion
          fetch-depth: 1
```

Reply:
Oh, I see.
I lived in the days when we did git fetch in script.
Nowadays it is done by tweaking in `actions/checkout`.